src/modules/ipops/ipops_mod.c

```diff
@@ -1292,6 +1297,31 @@ static int ki_dns_query(sip_msg_t *msg, str *naptrname, str *pvid)
 	return dns_update_pv(naptrname, pvid);
 }
 
+/**
+ *
```

Comment:
There is room for a description of the function here - that's why we have comments before functions.

Reply:
Fixed